### ISSUE TITLE
Review backlog of accessibility issues from VA.gov-team repo

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#9970]
## Description
The [accessibility backlog](https://github.com/department-of-veterans-affairs/va.gov-team/issues?q=is%3Aopen+is%3Aissue+label%3A508%2FAccessibility) within the VA.gov-team repo contains all reported accessibility issues the Platform team has found. Some of these issues are referencing products/features that are not the responsibility of Sitewide Crew, some issues may be stale and no longer issues or no longer be reproduced.

### Additional Context
Process if an issue is determined to no longer exist:

1. Add label `stale`
2. Add a comment and tag either (depending on the situation and/or who is left remaining on the project):
   1. Original reporter
   2. Assignee
   3. Last person that commented on the issue
3. Indicate that the issue cannot be reproduced or is no longer relevant and will be closed in 1 week unless otherwise indicated. 

## Acceptance Criteria
- [ ] Determine which issues in the backlog are the responsibility of Sitewide Crew and label appropriately
- [ ] Determine which of those issues still persist. If it is old/cannot be reproduce/etc. follow process above

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by EWashb** [comment]
this was labeled (by me) for sprint 64 as an error. This was never planned for sprint 64 and is still in the backlog.

**Comment by EWashb** [comment]
Give timeline for ticket closure once it's determined that you cannot reproduce.
